app/Http/Controllers/ConferencesController.php

```diff
@@ -8,7 +8,6 @@ class ConferencesController extends Controller
 {
     public function show(Conference $conference)
     {
-        return view('conferences.show')
-            ->with('conference', $conference);
```

Comment:
I prefer to be more explicit; I know it's repetitive, but a lot of people aren't familiar with `compact`. I don't have a *really* good justification for it, but it's more like a personal preference kind of thing.